server/mlabns/util/production_check.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+def is_production_site(site_name):
+    """Determines if the given site name matches the production site schema
+
+    Args:
+        site_name: Name of site to check (e.g. "lga01")
+
+    Returns:
+        True if the site name matches the schema of a production site.
+    """
+    return re.match('^[a-z]{3}\d{2}$', site_name, re.IGNORECASE) != None
```

Comment:
Should this be '^[a-zA-Z{3}\d{2}$'?  In the unit test you test for TUN01, noting that all caps are also valid.